src/Raven.Studio/typescript/viewmodels/database/indexes/clearIndexErrorsConfirm.ts

```diff
@@ -0,0 +1,59 @@
+import dialog = require("plugins/dialog");
+import database = require("models/resources/database");
+import dialogViewModelBase = require("viewmodels/dialogViewModelBase");
+import messagePublisher = require("common/messagePublisher");
+import clearIndexErrorsCommand = require("commands/database/index/clearIndexErrorsCommand");
+
+class clearIndexErrorsConfirm extends dialogViewModelBase {
+    title: string;
+    subTitleHtml: string;
+    clearErrorsTask = $.Deferred<boolean>();
+    indexesToClear = ko.observableArray<string>();
+    clearAllIndexes = ko.observable<boolean>();
+    
+    constructor(indexesToClear: string[], private db: database, private shouldClearAllIndexes: boolean) {
```

Comment:
I think shouldClearAll is redundant here. When we pass empty list then it means clear all.

Reply:
done